release.py

```diff
@@ -0,0 +1,289 @@
+        print(f"{bcolors.OKCYAN}Skipping clone step...{bcolors.ENDC}")
+    else:
+        try:
+            subprocess.run(
+                [
+                    "git",
+                    "clone",
+                    "https://github.com/tahoe-lafs/tahoe-lafs.git",
+                    RELEASE_FOLDER,
+                ],
+                check=True,
+            )
+            record_step("clone_complete")
+        except Exception as e:
+            print(f"{bcolors.FAIL}INFO: Failed to clone! :(...{bcolors.ENDC}")
+            print(f"{bcolors.FAIL} {e} {bcolors.ENDC}")
+    os.chdir(RELEASE_FOLDER)
+    if step_complete("install_deps_on_venv_complete"):
+        print(
+            f"{bcolors.OKCYAN}Skipping venv setup and dependency installation...{bcolors.ENDC}"
+        )
+    else:
+        try:
+            subprocess.run(["python", "-m", "venv", "venv"], check=True)
+            subprocess.run(
+                ["./venv/bin/pip", "install", "--editable", ".[test]"], check=True
+            )
+            record_step("install_deps_on_venv_complete")
+        except Exception as e:
+            print(
+                f"{bcolors.FAIL}INFO: Failed to install virtualenv and and dependencies clone! :(...{bcolors.ENDC}"
+            )
+            print(f"{bcolors.FAIL} {e} {bcolors.ENDC}")
+    if step_complete("branch_complete"):
+        print(f"{bcolors.OKCYAN}Skipping create release branch...{bcolors.ENDC}")
+    else:
+        try:
+            subprocess.run(["git", "branch", BRANCH], check=True)
+            record_step("branch_complete")
+        except Exception as e:
+            print(
+                f"{bcolors.FAIL}INFO: Failed to clean release branch! :(...{bcolors.ENDC}"
+            )
+            print(f"{bcolors.FAIL} {e} {bcolors.ENDC}")
+    subprocess.run(["git", "checkout", BRANCH])
+    if step_complete("tox_news_complete"):
+        print(f"{bcolors.OKCYAN}Skipping news generation...{bcolors.ENDC}")
+    else:
+        try:
+            subprocess.run(["./venv/bin/tox", "-e", "news"], check=True)
+            record_step("tox_news_complete")
+        except Exception as e:
+            print(f"{bcolors.FAIL}INFO: Failed to generate news! :(...{bcolors.ENDC}")
+            print(f"{bcolors.FAIL} {e} {bcolors.ENDC}")
+    if step_complete("newsfragment_complete"):
+        print(f"{bcolors.OKCYAN}Skipping add news fragment...{bcolors.ENDC}")
+    else:
+        try:
+            mkfile("newsfragments/{ticket}.minor".format(ticket=TICKET))
+            record_step("newsfragment_complete")
+        except Exception as e:
+            print(
+                f"{bcolors.FAIL}INFO: Failed to add newsfragment file! :(...{bcolors.ENDC}"
+            )
+            print(f"{bcolors.FAIL} {e} {bcolors.ENDC}")
+    if step_complete("commit_newsfragment_complete"):
+        print(f"{bcolors.OKCYAN}Skipping commit fragment...{bcolors.ENDC}")
+    else:
+        try:
+            subprocess.run(["git", "add", "."], check=True)
```

Comment:
`["git", "add", "newsfragments/{ticket}.minor".format(ticket=TICKET), "NEWS.txt"]` or so instead please (i.e. be explicit about the file(s) added).